opentelemetry-sdk/tests/metrics/export/test_export.py

```diff
@@ -69,7 +70,7 @@ def test_aggregator_for_counter(self):
     # TODO: Add other aggregator tests
 
     def test_checkpoint_set(self):
-        meter = metrics.Meter()
+        meter = metrics.MeterProvider().get_meter(__name__)
```

Comment:
I think these should all be switched to `metrics.get_meter(__name__)` People may look at docs for inspiration, and I think we should use the most simple / intuitive API.

Reply:
It would be the metrics api package that can do `get_meter`, so a meter provider would have to be added for these tests. I want to isolate the behavior to solely test specific small chunks of logic (like checkpoint set) so I don't want to include a dependency on the instrumentation behavior.